### PR TITLE
Refactor KafkaSink

### PR DESCRIPTION
**Pull request by mejroslav**
- There is no default topic name. Runtime Error is raised when topic or bootstrap.servers are not provided.
- `poll` function removed from `on_tick`
- It is possible to unthrottle the pipeline in `process`

### CODE REVIEW

**Review by PremyslCerny — COMMENTED** [review]
Please add comments to each change, like why we need the on tick etc.

File: bspump/kafka/sink.py [hunk 1]
```diff
@@ -173,17 +170,24 @@ async def _on_stop(self, event_name, pipeline):
...
+	async def _on_tick(self, event_name):
+		"""
+		Fallback mechanism for unthrottling the pipeline.
+		"""
+		if self.IsThrottling and (len(self.Producer) < self.LowWatermark):
+			# Unthrottle the pipeline
+			self.IsThrottling = False
+			self.Pipeline.throttle(self, False)
 
+	def _kafka_poll(self):
```
Comment (PremyslCerny) [inline bspump/kafka/sink.py:188]: Add unthrottle back to kafka_poll.

File: bspump/kafka/sink.py [hunk 2]
```diff
@@ -62,14 +60,16 @@ def process(self, context, event):
...
-		"watermark.low": "40000",
-		"watermark.high": "90000",
+		"topic": "",
 		"batch.num.messages": "100000",
 		"linger.ms": "500",  # This settings makes a significant impact on the throughtput
 		"batch.size": "1000000",
 		"poll.timeout": "0.1",
 		# "compression.type": "snappy",
+
+		# Custom options
```
Comment (ateska) [inline bspump/kafka/sink.py:70]: Options that are not for librdkafka ...

**Review by ateska — CHANGES_REQUESTED** [review]
Change in the comment.

Also, please test a throughput before merging.

**Pull request closed without merging**